libs/ledgerjs/packages/hw-app-eth/tests/EIP712/utils.unit.test.ts

```diff
@@ -0,0 +1,541 @@
+import BigNumber from "bignumber.js";
```

Comment:
Nice test but the pain should've been really hard 🥲

Reply:
It was. 🤣